setup.py

```diff
@@ -51,17 +51,16 @@ def find_version(*file_paths):
     'markupsafe==1.1.*',
```

Comment:
I don't think we are using this module, is it listed here to allow us to pin a dependency of a dependency?

Reply:
I **think** we used to have this in the old `lib` folder of Cylc... and I just copied it to setup.py (trying to remember)

Reply:
I think you are right there.

Reply:
It may have been to do with the old GUI.

Reply:
Feel free to remove if not used :+1: this change looks good, approving assuming CI passes.

Reply:
I pip uninstalled and the unittests still passed so I think we are good to drop it.